### DISCUSSION TITLE
韓國月球探測器8月帶BTS〈Dynamite〉MV上太空，有望成為全球第7個探月國家

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#438] [科技新知]
「韓國政府花費7年、耗資約2367億韓元（約合新台幣55億5298萬元）經費開發Danuri號，將成為繼日本、印度、歐盟、美國、俄羅斯及中國之後，第7個展開月球探索任務的國家。」
這真的要足夠的資本跟研究的整合才能達成，台灣現階段應該很難，還有很長的路要走。

資料來源：https://www.thenewslens.com/article/167961?_gl=1*j28rbw*_ga*MTJWNUY1YjJEU1BRaGZueWRLRUN4X1U3bHF0SEhPT1pBa2NyVlk2Szl5SmxMZnJ0UFJpTkxsOVU1T2xOSElGMg..

**Comment by YuHengChen54** [comment]
台灣目前要發射到太空的技術感覺還落後國外很多，真的還有很長一段路要走~~

**Comment by emily40533** [comment]
探索月球的任務也可以用BTS的曲子做噱頭宣傳，南韓在軟實力的方面真的很強🤣
不過他們現在也有發射到太空的硬實力了！

**Comment by number32984** [comment]
超英趕美，希望台灣也能成功

**Comment by NcuJerryKao** [comment]
中國已經成功發射好幾次太空梭到月球 這消息我當初 甚是驚訝
今天看到 竟然連韓國也都上去了 傻眼 想想日本、韓國、中國都上去了，那台灣呢 抱歉還差遠了

不過中央地科系 能預測地震事還蠻令人期待的 畢竟地科都說地震無法預測 抱歉扯遠了

**Comment by Ben6uo** [comment]
外星人聽到會不會攻打地球阿?

**Comment by todoroki0910** [comment]
如果我是外星人應該比較想看twice的mv

**Comment by skyling0299** [comment]
雖然帶BTS是噱頭，但這代表全球在太空的競爭逐漸增強，實力也越來越好

**Comment by brian431** [comment]
說到月球，美國最近也開始籌備要再次登月了。真好奇這次會有什麼新的發展。

**Comment by xCk27x** [comment]
台灣要能夠發射火箭還要很多年
目前還是以spaceX或其他公司來代為發射衛星
不過這幾年也有何國外合作的火星探測任務
可以期待一下 (但還要很久啦XD)

**Comment by limitx0** [comment]
臺灣的TASA也一直有在徵才
期待未來的發展

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by Aerofe** [comment]
現在商業策略越來越多樣化了，拍個MV也沒必要到月球😂

**Comment by awehao** [comment]
哈哈哈展現國家硬實力的同時也展現軟實力，希望台灣也可以做到有硬實力的同時也有軟實力

**Comment by Raywei0214** [comment]
到底為什麼是BTS，為什麼不是TFB青春修練手冊

**Comment by Yilin20021016** [comment]
我很企盼臺灣把五月天送上月球的那一天

**Comment by HsuChen88** [comment]
身為一名中央大學學生，希望中央大學有朝一日也能帶我們讓月球。

**Comment by ghost** [comment]
對於太空系的學生來說，看到目前越來越多國家積極開發太空這塊新商機真的很熱血沸騰，雖然台灣還有很長一段路需要走，但希望未來台灣也能成為太空產業大國。

**Comment by bww0401** [comment]
我覺得外星人聽到會發火吧(誤，據我所知，日前國家太空中心NSPO改名叫TASA，以增加國際間知名度，感覺台灣有在重視該領域的發展，希望台灣能早日登上月球
